test/fixtures/rules/std_rule_cases/L010.yml

```diff
@@ -177,3 +177,11 @@ test_fail_bigquery_week:
     rules:
       L010:
         capitalisation_policy: upper
+
+test_pass_ignore_schema:
+  pass_str: |
+    SELECT ignoreme.AhAhAhAh FROM dbo
+  configs:
+    rules:
+      L010:
+        ignore_schemas: ignoreme
```

Comment:
I don't see how this test is a valid test. L010 is about capitalisation of keywords but `ignoreme` is not a keyword.

Reply:
The idea was - and maybe I misunderstood something - that AhAhAhAh is not changed as it is in the schema that I want to ignore.

Reply:
This test should be in L014.yml as that's the rule that deals with capitalisation of unquoted identifiers, not in L010.yml (which deals with capitalisation of keywords).